Marlin/Marlin_main.cpp

```diff
@@ -1725,7 +1720,7 @@ static void clean_up_after_endstop_or_probe_move() {
                yy = y && !axis_homed[Y_AXIS],
                zz = z && !axis_homed[Z_AXIS];
     if (xx || yy || zz) {
-      SERIAL_ECHO_START;
+      SERIAL_ERROR_START;
```

Comment:
There is still a problem.
If the deploy was not scuccse full - we continue - and smash the nozzle into the bed. Throwing an error here does not help much. G28/G29/M48 do continue - only the host stops sending.